docs/mp/guides/05_metrics.adoc

```diff
@@ -166,6 +184,19 @@ curl -H "Accept: application/json"  http://localhost:8080/metrics/vendor/grpc.re
 
 NOTE: You cannot get the individual fields of a metric. For example, you cannot target http://localhost:8080/metrics/vendor/grpc.requests.meter.count.
 
+==== Controlling `REST.request` metrics
+Helidon implements the optional family of metrics, all with the name `REST.request`, as described in the
```

Comment:
Maybe a link here?

Reply:
Sigh. I had started to include one and got distracted and forgot to. Adding it.